index.html

```diff
@@ -16,64 +16,107 @@
       <div class="main-card">
         <h2 class="main-card__title">Question:</h2>
         <div class="main-card__content">
+          <i class="fas fa-bookmark"></i>
           <p>Was ist das chemische Symbol für Silber?</p>
           <details>
             <summary>Show Answer</summary>
             <p>Ag</p>
           </details>
           <ul class="main-card__answer">
-            <a href="#" class="main-card__answeritem"><li>Au</li></a>
-            <a href="#" class="main-card__answeritem"><li>Ag</li></a>
-            <a href="#" class="main-card__answeritem"><li>Pb</li></a>
-            <a href="#" class="main-card__answeritem"><li>Hg</li></a>
+            <a href="#" class="button main-card__answeritem"><li>Au</li></a>
```

Comment:
Only `li`-elements should be direct children of a list. Also, anchor elements are hyperlinks and always indicate a link to another page. You can read more about it here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a

One of their class names is `button`, however these are not `button`-elements. This would make more sense: 
```suggestion
            <li class="button main-card__answeritem"><button>Au</button></li>
```
You could also skip the `button`-element and style the `li`-element like a button (the styles in your class `button` would then be in the class `main-card__answeritem`) :
```suggestion
            <li class="main-card__answeritem">Au</li>
```